tools/identity-resolution/identity-resolution/Helpers/GitHubAadConverter.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Net.Http;
+using Azure.Core;
+using Azure.Identity;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Azure.Sdk.Tools.NotificationConfiguration.Helpers
+{
+    public class GitHubToAadConverter
```

Comment:
```suggestion
    public class GitHubToAADConverter
```

Usually acronyms are capital.